deploy/cephfs/helm/templates/nodeplugin-daemonset.yaml

```diff
@@ -142,6 +144,10 @@ spec:
         - name: ceph-csi-config
           configMap:
             name: {{ .Values.configMapName | quote }}
+        - name: keys-tmp-dir
+          emptyDir: {
```

Comment:
as this increases the file operation performance, will this cause any bottleneck for serving more number for requests? as this affects the container memory.

Reply:
Here is my **theoretical** reasoning on the footprint we would use,
- Number of requests in flight, depends on number of threads by the various side cars, seem to be configured at 100 (and I do not see a need to increase these yet)
- Hence we would use up, 100*(say 3)files*(say 2) sidecars in all, which is about 600, round it off to 1KiB (1024 files)
- 1KiB files with content as the base64 encoded Ceph key of 40 bytes each, should take up an inode each. Assuming an inode is 256 bytes this should take 256KiB in memory

I would hence assume this is fine and not a concern from a memory footprint standpoint.

Reply:
cc @humblec